app/api/currentuser/send_group_request.go

```diff
@@ -6,6 +6,42 @@ import (
 	"github.com/France-ioi/AlgoreaBackend/app/service"
 )
 
+// swagger:operation POST /current-user/group-requests/{group_id} groups users groupRequestCreate
+// ---
+// summary: Send a request to join a group
```

Comment:
Let's use "create a request" instead of "send a request" (x2) so that the reader does not think there is email sent or whatever.

Reply:
fixed